src/index.js

```diff
@@ -28,6 +28,48 @@ export function substring(str, begin = 0, end) {
     return str.match(astralRange).slice(begin, end).join("");
 }
 
+export function substr(str, begin = 0, len) {
+    // Check for input
+    if (typeof str !== "string") {
+        throw new Error("Input must be a string");
+    }
+
+    const strLength = length(str);
+
+    // Fix type
+    if (typeof begin !== "number") {
+        begin = parseInt(begin);
+    }
+
+    // Return zero-length string if got oversize number.
+    if(begin >= strLength){
+        return "";
+    }
+    // Calculating postive version of negative value.
+    else if(begin < 0) {
```

Comment:
Since the previous `if` returns something, this doesn't need to be `else if` and a simple `if` is enough. Also space needed between `if` and `(`. Please use ESLint to see all the styling issues.